### ISSUE TITLE
resp.Request.TraceInfo().TotalTime giving invalid time format

### ISSUE THREAD

**Issue by nitiprabhu** [go-resty/resty#350]
client := resty.New()

resp, err := client.R().
		EnableTrace().
		Get("https://httpbin.org/get")

With the above example:

I am using _resp.Request.TraceInfo().TotalTime_ to print the total time spent on the request. 
In the logs, most of the time I can see the time in the strange format, something like this: 

Time Took - is 2562047h47m16.854775807s
Time Took - is 2562047h47m16.854775807s

This is the same issue for method: resp.Time()

My Go Version: go version go1.13.10 darwin/amd64

Could anybody please help me understand what went wrong here?

**Comment by moorereason** [comment]
What version of resty are you using?

**Comment by jeevatkm** [comment]
@nitiprabhu Can you let us know the resty version you're using?

**Comment by nitiprabhu** [comment]
@moorereason @jeevatkm sorry for the delay, the version is: **v2.3.0**

**Comment by moorereason** [comment]
I can't reproduce this.  Can you provide a test case that reproduces the issue?

**Comment by jeevatkm** [comment]
@nitiprabhu Thanks for the response. @moorereason have tried to reproduce it, no luck.
Let's understand the problem first and come to the same page.

What you're trying to do and how you're measuring, do you mind describing in detail?
E.g.: you have some local or remote server endpoints accessing via resty and capturing the trace info.

Also, if you could create a Go test case to demonstrate an issue, would be helpful.

**Comment by nitiprabhu** [comment]
@jeevatkm @moorereason This can be produced when you re-use the existing resty-client to send the requests.
**Example:** 

```
package main

import (
	"fmt"
	resty "github.com/go-resty/resty/v2"
)

func main() {
	restyClient := resty.New().SetRetryCount(1).EnableTrace()
	for i := 0; i < 1000; i++ {
		r, e := restyClient.R().Get("https://httpbin.org/get")
		if e == nil {
			v := r.Request.TraceInfo().TotalTime
			fmt.Println(v)
		}
	}
}
```

**Comment by jeevatkm** [comment]
@nitiprabhu Thanks for the sample code snippet to describe the issue.

I was able to find the reference, there was an issue in total time when connection was reused. @moorereason have already addressed this issue in #347. It will be released in the upcoming v2.4.0

You could try out and share feedback.
```
# perform this go get 
go get github.com/go-resty/resty/v2@master

# OR add below line in your go.mod
github.com/go-resty/resty/v2 v2.3.1-0.20200904223602-d2ba00f5a573
```

**Comment by nitiprabhu** [comment]
@jeevatkm Tried, it is working fine here (v2.3.1-0.20200904223602-d2ba00f5a573). Looking forward to using version v2.4.0
